frontend/src/app/components/wp-card-view/event-handler/click-handler.ts

```diff
@@ -0,0 +1,72 @@
+import {Injector} from '@angular/core';
+import {CardEventHandler} from "core-components/wp-card-view/event-handler/card-view-handler-registry";
+import {WorkPackageCardViewComponent} from "core-components/wp-card-view/wp-card-view.component";
+import {WorkPackageTableSelection} from "core-components/wp-fast-table/state/wp-table-selection.service";
+import {WorkPackageTableFocusService} from "core-components/wp-fast-table/state/wp-table-focus.service";
+import {WorkPackageCardViewService} from "core-components/wp-card-view/services/wp-card-view.service";
+
+export class CardClickHandler implements CardEventHandler {
```

Comment:
This class too is almost identical to the table counterpart. Again we would benefit from subclassing them.